samples/bluetooth/broadcast_audio_sink/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf

```diff
@@ -1,3 +1,16 @@
 # The LC3 codec uses a large amount of stack. This app runs the codec in the work-queue, hence
 # inctease stack size for that thread.
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+# Enable encryption.
+CONFIG_BT_TINYCRYPT_ECC=y
+# LC3 Needs these
+CONFIG_BT_ISO_RX_BUF_COUNT=20
+CONFIG_BT_ISO_SYNC_RECEIVER=y
+CONFIG_BT_ISO_MAX_CHAN=5
+CONFIG_BT_ISO_RX_MTU=60
+#Enable USB
+CONFIG_USE_USB_AUDIO_OUTPUT=y
+CONFIG_USB_DEVICE_PRODUCT="USB Broadcast Sink"
```

Comment:
> Not sure why I would get a warning?

Because you are setting a value that depends on another Kconfig that is optional. 

For example, if I build with `CONFIG_ENABLE_LC3=n` then I get the following warnings:
```
warning: USE_USB_AUDIO_OUTPUT (defined at
/home/thalley/dev/nordic/zephyrproject/zephyr/samples/bluetooth/broadcast_audio_sink/Kconfig:46) was
assigned the value 'y' but got the value 'n'. Check these unsatisfied dependencies: ENABLE_LC3 (=n).
See http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_USE_USB_AUDIO_OUTPUT and/or look up
USE_USB_AUDIO_OUTPUT in the menuconfig/guiconfig interface. The Application Development Primer,
Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be
helpful too.


warning: USB_DEVICE_PRODUCT (defined at subsys/usb/device/Kconfig:37) was assigned the value 'USB
Broadcast Sink' but got the value ''. Check these unsatisfied dependencies: USB_DEVICE_STACK (=n).
See http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_USB_DEVICE_PRODUCT and/or look up
USB_DEVICE_PRODUCT in the menuconfig/guiconfig interface. The Application Development Primer,
Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be
helpful too.
```

That might be OK since it's enabled by default though

Reply:
I think I will add `CONFIG_ENABLE_LC3=y` in this conf file as well to make this clearer.